kong/plugins/syslog/handler.lua

```diff
@@ -20,32 +20,52 @@ local LOG_LEVELS = {
   emerg = 0
 }
 
+local FACILITIES = {
+  AUTH = lsyslog.FACILITY_AUTH,
+  AUTHPRIV = lsyslog.FACILITY_AUTHPRIV,
+  CRON = lsyslog.FACILITY_CRON,
+  DAEMON = lsyslog.FACILITY_DAEMON,
+  FTP = lsyslog.FACILITY_FTP,
+  KERN = lsyslog.FACILITY_KERN,
+  LPR = lsyslog.FACILITY_LPR,
+  MAIL = lsyslog.FACILITY_MAIL,
+  NEWS = lsyslog.FACILITY_NEWS,
+  SYSLOG = lsyslog.FACILITY_SYSLOG,
+  USER = lsyslog.FACILITY_USER,
+  UUCP = lsyslog.FACILITY_UUCP,
+  LOCAL0 = lsyslog.FACILITY_LOCAL0,
+  LOCAL1 = lsyslog.FACILITY_LOCAL1,
+  LOCAL2 = lsyslog.FACILITY_LOCAL2,
+  LOCAL3 = lsyslog.FACILITY_LOCAL3,
+  LOCAL4 = lsyslog.FACILITY_LOCAL4,
+  LOCAL5 = lsyslog.FACILITY_LOCAL5,
+  LOCAL6 = lsyslog.FACILITY_LOCAL6,
+  LOCAL7 = lsyslog.FACILITY_LOCAL7
+}
 
-local function send_to_syslog(log_level, severity, message)
+local function send_to_syslog(log_level, severity, message, facility)
   if LOG_LEVELS[severity] <= LOG_LEVELS[log_level] then
-    lsyslog.open(SENDER_NAME, lsyslog.FACILITY_USER)
+    lsyslog.open(SENDER_NAME, FACILITIES[facility])
     lsyslog.log(lsyslog["LOG_" .. upper(severity)], cjson.encode(message))
   end
 end
 
-
 local function log(premature, conf, message)
   if premature then
     return
   end
 
   if message.response.status >= 500 then
-    send_to_syslog(conf.log_level, conf.server_errors_severity, message)
+    send_to_syslog(conf.log_level, conf.server_errors_severity, message, conf.syslog_facility)
 
   elseif message.response.status >= 400 then
-    send_to_syslog(conf.log_level, conf.client_errors_severity, message)
+    send_to_syslog(conf.log_level, conf.client_errors_severity, message, conf.syslog_facility)
 
   else
-    send_to_syslog(conf.log_level, conf.successful_severity, message)
+    send_to_syslog(conf.log_level, conf.successful_severity, message, conf.syslog_facility)
   end
 end
 
```

Comment:
```suggestion


```
code style is 2 lines vertical white space between functions, please do not delete those